powershell/upload/UploadFile.ps1

```diff
@@ -80,17 +80,24 @@ catch {
 $fileApiClient = [FileApiClient]::new($config.Services.FileApiBaseUrl, $token)
 $fileApiService = [FileApiService]::new($fileApiClient, $config.Upload.TenantId, $config.Upload.BusinessTypeId)
 
-#region Upload file
+#region Upload Directory contents
 
-try {
-    $createdFilePath = $fileApiService.CreateFileToUpload($config.Upload.Path) 
-    $fileApiService.UploadFile($createdFilePath, $(Split-Path -Path $config.Upload.Path -Leaf))
-}
-catch {
-    [Helper]::EndProgramWithError($_, "Failure uploading the file.")
+Get-ChildItem -Path $config.Upload.Path -Filter $config.Upload.Filter | ForEach-Object -Process {
+
+    try {
+        $createdFilePath = $fileApiService.CreateFileToUpload($_.FullName) 
+        $fileApiService.UploadFile($createdFilePath, $(Split-Path -Path $_.FullName -Leaf))
+        $archivedFile =  [Helper]::ArchiveFile($config.Upload.ArchivePath, $_.FullName)
+        if(-not [string]::IsNullOrEmpty($archivedFile)){
+            Write-Host "File archived ($($archivedFile))."
+        }
+    }
+    catch {
+        [Helper]::EndProgramWithError($_, "Failure uploading the file.")
```

Comment:
Is it possible that the error message contains the name of the file?

Reply:
Added filename to errormessage